dendrite-config.yaml

```diff
@@ -0,0 +1,299 @@
+# This is the Dendrite configuration file.
+#
+# The configuration is split up into sections - each Dendrite component has a
+# configuration section, in addition to the "global" section which applies to
+# all components.
+#
+# At a minimum, to get started, you will need to update the settings in the
+# "global" section for your deployment, and you will need to check that the
+# database "connection_string" line in each component section is correct. 
+#
+# Each component with a "database" section can accept the following formats
+# for "connection_string":
+#   SQLite:     file:filename.db
+#               file:///path/to/filename.db
+#   PostgreSQL: postgresql://user:pass@hostname/database?params=...
+#
+# SQLite is embedded into Dendrite and therefore no further prerequisites are
+# needed for the database when using SQLite mode. However, performance with
+# PostgreSQL is significantly better and recommended for multi-user deployments.
+#
+# The "max_open_conns" and "max_idle_conns" settings configure the maximum 
+# number of open/idle database connections. The value 0 will use the database
+# engine default, and a negative value will use unlimited connections. The
+# "conn_max_lifetime" option controls the maximum length of time a database
+# connection can be idle in seconds - a negative value is unlimited.
+
+# The version of the configuration file. 
+version: 1
+
+# Global Matrix configuration. This configuration applies to all components.
+global:
+  # The domain name of this homeserver.
+  server_name: localhost
+
+  # The path to the signing private key file, used to sign requests and events.
+  private_key: matrix_key.pem
+
+  # A unique identifier for this private key. Must start with the prefix "ed25519:".
+  key_id: ed25519:auto
+
+  # How long a remote server can cache our server signing key before requesting it
+  # again. Increasing this number will reduce the number of requests made by other
+  # servers for our key but increases the period that a compromised key will be
+  # considered valid by other homeservers.
+  key_validity_period: 168h0m0s
+
+  # Lists of domains that the server will trust as identity servers to verify third
+  # party identifiers such as phone numbers and email addresses.
+  trusted_third_party_id_servers:
+  - matrix.org
+  - vector.im
+
+  # Configuration for Kafka/Naffka.
+  kafka:
+    # List of Kafka addresses to connect to. This is not needed if using Naffka.
+    addresses: []
```

Comment:
We should mention the word 'broker' somewhere, and maybe give an example using the default broker port.

Reply:
We can probably safely set a default value here to `localhost:2181` - it should get safely ignored since naffka is enabled by default anyway.